Creating CheckRun fails with annotation validation error

The error is stated in the Title.
Repo Run: https://github.com/alvr-org/PhoneVR/actions/runs/10678178487
Repo: https://github.com/alvr-org/PhoneVR
PR: https://github.com/alvr-org/PhoneVR/pull/376

Full Error:
```
  2024-09-03 07:59:39 +0000 - publish - DEBUG - action triggered by 'pull_request' event
  2024-09-03 07:59:39 +0000 - publish - DEBUG - Settings: Settings(token='***', actor='github-actions', api_url='https://api.github.com/', graphql_url='https://api.github.com/graphql', api_retries=10, event={'action': 'synchronize', 'after': '2ea57f71dfdd7c9c5be44eba3fa31a15667d4973', 'before': '3d01fcf1cc198e24b45cdc610dd0735151a047ed', 'number': 376, 'organization': {'avatar_url': 'https://avatars.githubusercontent.com/u/74990209?v=4', 'description': '', 'events_url': 'https://api.github.com/orgs/alvr-org/events', 'hooks_url': 'https://api.github.com/orgs/alvr-org/hooks', 'id': 74990209, 'issues_url': 'https://api.github.com/orgs/alvr-org/issues', 'login': 'alvr-org', 'members_url': '[https://api.github.com/orgs/alvr-org/members{/member}](https://api.github.com/orgs/alvr-org/members%7B/member%7D)', 'node_id': 'MDEyOk9yZ2FuaXphdGlvbjc0OTkwMjA5', 'public_members_url': '[https://api.github.com/orgs/alvr-org/public_members{/member}](https://api.github.com/orgs/alvr-org/public_members%7B/member%7D)', 'repos_url': 'https://api.github.com/orgs/alvr-org/repos', 'url': 'https://api.github.com/orgs/alvr-org'}, 'pull_request': {'_links': {'comments': {'href': 'https://api.github.com/repos/alvr-org/PhoneVR/issues/376/comments'}, 'commits': {'href': 'https://api.github.com/repos/alvr-org/PhoneVR/pulls/376/commits'}, 'html': {'href': 'https://github.com/alvr-org/PhoneVR/pull/376'}, 'issue': {'href': 'https://api.github.com/repos/alvr-org/PhoneVR/issues/376'}, 'review_comment': {'href': '[https://api.github.com/repos/alvr-org/PhoneVR/pulls/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/pulls/comments%7B/number%7D)'}, 'review_comments': {'href': 'https://api.github.com/repos/alvr-org/PhoneVR/pulls/376/comments'}, 'self': {'href': 'https://api.github.com/repos/alvr-org/PhoneVR/pulls/376'}, 'statuses': {'href': 'https://api.github.com/repos/alvr-org/PhoneVR/statuses/2ea57f71dfdd7c9c5be44eba3fa31a15667d4973'}}, 'active_lock_reason': None, 'additions': 284, 'assignee': None, 'assignees': [], 'author_association': 'MEMBER', 'auto_merge': None, 'base': {'label': 'alvr-org:master', 'ref': 'master', 'repo': {'allow_auto_merge': True, 'allow_forking': True, 'allow_merge_commit': True, 'allow_rebase_merge': True, 'allow_squash_merge': True, 'allow_update_branch': False, 'archive_url': '[https://api.github.com/repos/alvr-org/PhoneVR/{archive_format}{/ref}](https://api.github.com/repos/alvr-org/PhoneVR/%7Barchive_format%7D%7B/ref%7D)', 'archived': False, 'assignees_url': '[https://api.github.com/repos/alvr-org/PhoneVR/assignees{/user}](https://api.github.com/repos/alvr-org/PhoneVR/assignees%7B/user%7D)', 'blobs_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/blobs{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/blobs%7B/sha%7D)', 'branches_url': '[https://api.github.com/repos/alvr-org/PhoneVR/branches{/branch}](https://api.github.com/repos/alvr-org/PhoneVR/branches%7B/branch%7D)', 'clone_url': 'https://github.com/alvr-org/PhoneVR.git', 'collaborators_url': '[https://api.github.com/repos/alvr-org/PhoneVR/collaborators{/collaborator}](https://api.github.com/repos/alvr-org/PhoneVR/collaborators%7B/collaborator%7D)', 'comments_url': '[https://api.github.com/repos/alvr-org/PhoneVR/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/comments%7B/number%7D)', 'commits_url': '[https://api.github.com/repos/alvr-org/PhoneVR/commits{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/commits%7B/sha%7D)', 'compare_url': '[https://api.github.com/repos/alvr-org/PhoneVR/compare/{base}...{head}](https://api.github.com/repos/alvr-org/PhoneVR/compare/%7Bbase%7D...%7Bhead%7D)', 'contents_url': '[https://api.github.com/repos/alvr-org/PhoneVR/contents/{+path}](https://api.github.com/repos/alvr-org/PhoneVR/contents/%7B+path%7D)', 'contributors_url': 'https://api.github.com/repos/alvr-org/PhoneVR/contributors', 'created_at': '2020-07-22T17:13:03Z', 'default_branch': 'master', 'delete_branch_on_merge': False, 'deployments_url': 'https://api.github.com/repos/alvr-org/PhoneVR/deployments', 'description': 'Use Steam VR-enabled applications with your phone as HMD (Head-mounted display). The only Open-Source solution to similar commercial packages like VRidge, iVRy, Trinus etc etc.', 'disabled': False, 'downloads_url': 'https://api.github.com/repos/alvr-org/PhoneVR/downloads', 'events_url': 'https://api.github.com/repos/alvr-org/PhoneVR/events', 'fork': False, 'forks': 38, 'forks_count': 38, 'forks_url': 'https://api.github.com/repos/alvr-org/PhoneVR/forks', 'full_name': 'alvr-org/PhoneVR', 'git_commits_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/commits{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/commits%7B/sha%7D)', 'git_refs_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/refs{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/refs%7B/sha%7D)', 'git_tags_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/tags{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/tags%7B/sha%7D)', 'git_url': 'git://github.com/alvr-org/PhoneVR.git', 'has_discussions': False, 'has_downloads': True, 'has_issues': True, 'has_pages': False, 'has_projects': True, 'has_wiki': True, 'homepage': '', 'hooks_url': 'https://api.github.com/repos/alvr-org/PhoneVR/hooks', 'html_url': 'https://github.com/alvr-org/PhoneVR', 'id': 281738404, 'is_template': False, 'issue_comment_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues/comments%7B/number%7D)', 'issue_events_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues/events{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues/events%7B/number%7D)', 'issues_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues%7B/number%7D)', 'keys_url': '[https://api.github.com/repos/alvr-org/PhoneVR/keys{/key_id}](https://api.github.com/repos/alvr-org/PhoneVR/keys%7B/key_id%7D)', 'labels_url': '[https://api.github.com/repos/alvr-org/PhoneVR/labels{/name}](https://api.github.com/repos/alvr-org/PhoneVR/labels%7B/name%7D)', 'language': 'C++', 'languages_url': 'https://api.github.com/repos/alvr-org/PhoneVR/languages', 'license': {'key': 'gpl-3.0', 'name': 'GNU General Public License v3.0', 'node_id': 'MDc6TGljZW5zZTk=', 'spdx_id': 'GPL-3.0', 'url': 'https://api.github.com/licenses/gpl-3.0'}, 'merge_commit_message': 'PR_TITLE', 'merge_commit_title': 'MERGE_MESSAGE', 'merges_url': 'https://api.github.com/repos/alvr-org/PhoneVR/merges', 'milestones_url': '[https://api.github.com/repos/alvr-org/PhoneVR/milestones{/number}](https://api.github.com/repos/alvr-org/PhoneVR/milestones%7B/number%7D)', 'mirror_url': None, 'name': 'PhoneVR', 'node_id': 'MDEwOlJlcG9zaXRvcnkyODE3Mzg0MDQ=', 'notifications_url': '[https://api.github.com/repos/alvr-org/PhoneVR/notifications{?since,all,participating}](https://api.github.com/repos/alvr-org/PhoneVR/notifications%7B?since,all,participating})', 'open_issues': 32, 'open_issues_count': 32, 'owner': {'avatar_url': 'https://avatars.githubusercontent.com/u/74990209?v=4', 'events_url': '[https://api.github.com/users/alvr-org/events{/privacy}](https://api.github.com/users/alvr-org/events%7B/privacy%7D)', 'followers_url': 'https://api.github.com/users/alvr-org/followers', 'following_url': '[https://api.github.com/users/alvr-org/following{/other_user}](https://api.github.com/users/alvr-org/following%7B/other_user%7D)', 'gists_url': '[https://api.github.com/users/alvr-org/gists{/gist_id}](https://api.github.com/users/alvr-org/gists%7B/gist_id%7D)', 'gravatar_id': '', 'html_url': 'https://github.com/alvr-org', 'id': 74990209, 'login': 'alvr-org', 'node_id': 'MDEyOk9yZ2FuaXphdGlvbjc0OTkwMjA5', 'organizations_url': 'https://api.github.com/users/alvr-org/orgs', 'received_events_url': 'https://api.github.com/users/alvr-org/received_events', 'repos_url': 'https://api.github.com/users/alvr-org/repos', 'site_admin': False, 'starred_url': '[https://api.github.com/users/alvr-org/starred{/owner}{/repo}](https://api.github.com/users/alvr-org/starred%7B/owner%7D%7B/repo%7D)', 'subscriptions_url': 'https://api.github.com/users/alvr-org/subscriptions', 'type': 'Organization', 'url': 'https://api.github.com/users/alvr-org'}, 'private': False, 'pulls_url': '[https://api.github.com/repos/alvr-org/PhoneVR/pulls{/number}](https://api.github.com/repos/alvr-org/PhoneVR/pulls%7B/number%7D)', 'pushed_at': '2024-09-03T07:07:02Z', 'releases_url': '[https://api.github.com/repos/alvr-org/PhoneVR/releases{/id}](https://api.github.com/repos/alvr-org/PhoneVR/releases%7B/id%7D)', 'size': 24280, 'squash_merge_commit_message': 'COMMIT_MESSAGES', 'squash_merge_commit_title': 'COMMIT_OR_PR_TITLE', 'ssh_url': 'git@github.com:alvr-org/PhoneVR.git', 'stargazers_count': 546, 'stargazers_url': 'https://api.github.com/repos/alvr-org/PhoneVR/stargazers', 'statuses_url': '[https://api.github.com/repos/alvr-org/PhoneVR/statuses/{sha}](https://api.github.com/repos/alvr-org/PhoneVR/statuses/%7Bsha%7D)', 'subscribers_url': 'https://api.github.com/repos/alvr-org/PhoneVR/subscribers', 'subscription_url': 'https://api.github.com/repos/alvr-org/PhoneVR/subscription', 'svn_url': 'https://github.com/alvr-org/PhoneVR', 'tags_url': 'https://api.github.com/repos/alvr-org/PhoneVR/tags', 'teams_url': 'https://api.github.com/repos/alvr-org/PhoneVR/teams', 'topics': ['android', 'asio', 'eigen', 'hmd', 'jni', 'mobile-vr', 'opengl-es', 'phone', 'phone-hmd', 'phonevr', 'smartphone', 'steamvr', 'steamvr-driver', 'virtual-reality', 'vr', 'wifi'], 'trees_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/trees{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/trees%7B/sha%7D)', 'updated_at': '2024-09-03T02:01:39Z', 'url': 'https://api.github.com/repos/alvr-org/PhoneVR', 'use_squash_pr_title_as_default': False, 'visibility': 'public', 'watchers': 546, 'watchers_count': 546, 'web_commit_signoff_required': False}, 'sha': '5abee7763f14bc01eefb80c266f60e5d3de2a0a5', 'user': {'avatar_url': 'https://avatars.githubusercontent.com/u/74990209?v=4', 'events_url': '[https://api.github.com/users/alvr-org/events{/privacy}](https://api.github.com/users/alvr-org/events%7B/privacy%7D)', 'followers_url': 'https://api.github.com/users/alvr-org/followers', 'following_url': '[https://api.github.com/users/alvr-org/following{/other_user}](https://api.github.com/users/alvr-org/following%7B/other_user%7D)', 'gists_url': '[https://api.github.com/users/alvr-org/gists{/gist_id}](https://api.github.com/users/alvr-org/gists%7B/gist_id%7D)', 'gravatar_id': '', 'html_url': 'https://github.com/alvr-org', 'id': 74990209, 'login': 'alvr-org', 'node_id': 'MDEyOk9yZ2FuaXphdGlvbjc0OTkwMjA5', 'organizations_url': 'https://api.github.com/users/alvr-org/orgs', 'received_events_url': 'https://api.github.com/users/alvr-org/received_events', 'repos_url': 'https://api.github.com/users/alvr-org/repos', 'site_admin': False, 'starred_url': '[https://api.github.com/users/alvr-org/starred{/owner}{/repo}](https://api.github.com/users/alvr-org/starred%7B/owner%7D%7B/repo%7D)', 'subscriptions_url': 'https://api.github.com/users/alvr-org/subscriptions', 'type': 'Organization', 'url': 'https://api.github.com/users/alvr-org'}}, 'body': None, 'changed_files': 37, 'closed_at': None, 'comments': 0, 'comments_url': 'https://api.github.com/repos/alvr-org/PhoneVR/issues/376/comments', 'commits': 7, 'commits_url': 'https://api.github.com/repos/alvr-org/PhoneVR/pulls/376/commits', 'created_at': '2024-09-03T07:01:43Z', 'deletions': 247, 'diff_url': 'https://github.com/alvr-org/PhoneVR/pull/376.diff', 'draft': False, 'head': {'label': 'alvr-org:test-ci-debug', 'ref': 'test-ci-debug', 'repo': {'allow_auto_merge': True, 'allow_forking': True, 'allow_merge_commit': True, 'allow_rebase_merge': True, 'allow_squash_merge': True, 'allow_update_branch': False, 'archive_url': '[https://api.github.com/repos/alvr-org/PhoneVR/{archive_format}{/ref}](https://api.github.com/repos/alvr-org/PhoneVR/%7Barchive_format%7D%7B/ref%7D)', 'archived': False, 'assignees_url': '[https://api.github.com/repos/alvr-org/PhoneVR/assignees{/user}](https://api.github.com/repos/alvr-org/PhoneVR/assignees%7B/user%7D)', 'blobs_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/blobs{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/blobs%7B/sha%7D)', 'branches_url': '[https://api.github.com/repos/alvr-org/PhoneVR/branches{/branch}](https://api.github.com/repos/alvr-org/PhoneVR/branches%7B/branch%7D)', 'clone_url': 'https://github.com/alvr-org/PhoneVR.git', 'collaborators_url': '[https://api.github.com/repos/alvr-org/PhoneVR/collaborators{/collaborator}](https://api.github.com/repos/alvr-org/PhoneVR/collaborators%7B/collaborator%7D)', 'comments_url': '[https://api.github.com/repos/alvr-org/PhoneVR/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/comments%7B/number%7D)', 'commits_url': '[https://api.github.com/repos/alvr-org/PhoneVR/commits{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/commits%7B/sha%7D)', 'compare_url': '[https://api.github.com/repos/alvr-org/PhoneVR/compare/{base}...{head}](https://api.github.com/repos/alvr-org/PhoneVR/compare/%7Bbase%7D...%7Bhead%7D)', 'contents_url': '[https://api.github.com/repos/alvr-org/PhoneVR/contents/{+path}](https://api.github.com/repos/alvr-org/PhoneVR/contents/%7B+path%7D)', 'contributors_url': 'https://api.github.com/repos/alvr-org/PhoneVR/contributors', 'created_at': '2020-07-22T17:13:03Z', 'default_branch': 'master', 'delete_branch_on_merge': False, 'deployments_url': 'https://api.github.com/repos/alvr-org/PhoneVR/deployments', 'description': 'Use Steam VR-enabled applications with your phone as HMD (Head-mounted display). The only Open-Source solution to similar commercial packages like VRidge, iVRy, Trinus etc etc.', 'disabled': False, 'downloads_url': 'https://api.github.com/repos/alvr-org/PhoneVR/downloads', 'events_url': 'https://api.github.com/repos/alvr-org/PhoneVR/events', 'fork': False, 'forks': 38, 'forks_count': 38, 'forks_url': 'https://api.github.com/repos/alvr-org/PhoneVR/forks', 'full_name': 'alvr-org/PhoneVR', 'git_commits_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/commits{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/commits%7B/sha%7D)', 'git_refs_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/refs{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/refs%7B/sha%7D)', 'git_tags_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/tags{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/tags%7B/sha%7D)', 'git_url': 'git://github.com/alvr-org/PhoneVR.git', 'has_discussions': False, 'has_downloads': True, 'has_issues': True, 'has_pages': False, 'has_projects': True, 'has_wiki': True, 'homepage': '', 'hooks_url': 'https://api.github.com/repos/alvr-org/PhoneVR/hooks', 'html_url': 'https://github.com/alvr-org/PhoneVR', 'id': 281738404, 'is_template': False, 'issue_comment_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues/comments%7B/number%7D)', 'issue_events_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues/events{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues/events%7B/number%7D)', 'issues_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues%7B/number%7D)', 'keys_url': '[https://api.github.com/repos/alvr-org/PhoneVR/keys{/key_id}](https://api.github.com/repos/alvr-org/PhoneVR/keys%7B/key_id%7D)', 'labels_url': '[https://api.github.com/repos/alvr-org/PhoneVR/labels{/name}](https://api.github.com/repos/alvr-org/PhoneVR/labels%7B/name%7D)', 'language': 'C++', 'languages_url': 'https://api.github.com/repos/alvr-org/PhoneVR/languages', 'license': {'key': 'gpl-3.0', 'name': 'GNU General Public License v3.0', 'node_id': 'MDc6TGljZW5zZTk=', 'spdx_id': 'GPL-3.0', 'url': 'https://api.github.com/licenses/gpl-3.0'}, 'merge_commit_message': 'PR_TITLE', 'merge_commit_title': 'MERGE_MESSAGE', 'merges_url': 'https://api.github.com/repos/alvr-org/PhoneVR/merges', 'milestones_url': '[https://api.github.com/repos/alvr-org/PhoneVR/milestones{/number}](https://api.github.com/repos/alvr-org/PhoneVR/milestones%7B/number%7D)', 'mirror_url': None, 'name': 'PhoneVR', 'node_id': 'MDEwOlJlcG9zaXRvcnkyODE3Mzg0MDQ=', 'notifications_url': '[https://api.github.com/repos/alvr-org/PhoneVR/notifications{?since,all,participating}](https://api.github.com/repos/alvr-org/PhoneVR/notifications%7B?since,all,participating})', 'open_issues': 32, 'open_issues_count': 32, 'owner': {'avatar_url': 'https://avatars.githubusercontent.com/u/74990209?v=4', 'events_url': '[https://api.github.com/users/alvr-org/events{/privacy}](https://api.github.com/users/alvr-org/events%7B/privacy%7D)', 'followers_url': 'https://api.github.com/users/alvr-org/followers', 'following_url': '[https://api.github.com/users/alvr-org/following{/other_user}](https://api.github.com/users/alvr-org/following%7B/other_user%7D)', 'gists_url': '[https://api.github.com/users/alvr-org/gists{/gist_id}](https://api.github.com/users/alvr-org/gists%7B/gist_id%7D)', 'gravatar_id': '', 'html_url': 'https://github.com/alvr-org', 'id': 74990209, 'login': 'alvr-org', 'node_id': 'MDEyOk9yZ2FuaXphdGlvbjc0OTkwMjA5', 'organizations_url': 'https://api.github.com/users/alvr-org/orgs', 'received_events_url': 'https://api.github.com/users/alvr-org/received_events', 'repos_url': 'https://api.github.com/users/alvr-org/repos', 'site_admin': False, 'starred_url': '[https://api.github.com/users/alvr-org/starred{/owner}{/repo}](https://api.github.com/users/alvr-org/starred%7B/owner%7D%7B/repo%7D)', 'subscriptions_url': 'https://api.github.com/users/alvr-org/subscriptions', 'type': 'Organization', 'url': 'https://api.github.com/users/alvr-org'}, 'private': False, 'pulls_url': '[https://api.github.com/repos/alvr-org/PhoneVR/pulls{/number}](https://api.github.com/repos/alvr-org/PhoneVR/pulls%7B/number%7D)', 'pushed_at': '2024-09-03T07:07:02Z', 'releases_url': '[https://api.github.com/repos/alvr-org/PhoneVR/releases{/id}](https://api.github.com/repos/alvr-org/PhoneVR/releases%7B/id%7D)', 'size': 24280, 'squash_merge_commit_message': 'COMMIT_MESSAGES', 'squash_merge_commit_title': 'COMMIT_OR_PR_TITLE', 'ssh_url': 'git@github.com:alvr-org/PhoneVR.git', 'stargazers_count': 546, 'stargazers_url': 'https://api.github.com/repos/alvr-org/PhoneVR/stargazers', 'statuses_url': '[https://api.github.com/repos/alvr-org/PhoneVR/statuses/{sha}](https://api.github.com/repos/alvr-org/PhoneVR/statuses/%7Bsha%7D)', 'subscribers_url': 'https://api.github.com/repos/alvr-org/PhoneVR/subscribers', 'subscription_url': 'https://api.github.com/repos/alvr-org/PhoneVR/subscription', 'svn_url': 'https://github.com/alvr-org/PhoneVR', 'tags_url': 'https://api.github.com/repos/alvr-org/PhoneVR/tags', 'teams_url': 'https://api.github.com/repos/alvr-org/PhoneVR/teams', 'topics': ['android', 'asio', 'eigen', 'hmd', 'jni', 'mobile-vr', 'opengl-es', 'phone', 'phone-hmd', 'phonevr', 'smartphone', 'steamvr', 'steamvr-driver', 'virtual-reality', 'vr', 'wifi'], 'trees_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/trees{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/trees%7B/sha%7D)', 'updated_at': '2024-09-03T02:01:39Z', 'url': 'https://api.github.com/repos/alvr-org/PhoneVR', 'use_squash_pr_title_as_default': False, 'visibility': 'public', 'watchers': 546, 'watchers_count': 546, 'web_commit_signoff_required': False}, 'sha': '2ea57f71dfdd7c9c5be44eba3fa31a15667d4973', 'user': {'avatar_url': 'https://avatars.githubusercontent.com/u/74990209?v=4', 'events_url': '[https://api.github.com/users/alvr-org/events{/privacy}](https://api.github.com/users/alvr-org/events%7B/privacy%7D)', 'followers_url': 'https://api.github.com/users/alvr-org/followers', 'following_url': '[https://api.github.com/users/alvr-org/following{/other_user}](https://api.github.com/users/alvr-org/following%7B/other_user%7D)', 'gists_url': '[https://api.github.com/users/alvr-org/gists{/gist_id}](https://api.github.com/users/alvr-org/gists%7B/gist_id%7D)', 'gravatar_id': '', 'html_url': 'https://github.com/alvr-org', 'id': 74990209, 'login': 'alvr-org', 'node_id': 'MDEyOk9yZ2FuaXphdGlvbjc0OTkwMjA5', 'organizations_url': 'https://api.github.com/users/alvr-org/orgs', 'received_events_url': 'https://api.github.com/users/alvr-org/received_events', 'repos_url': 'https://api.github.com/users/alvr-org/repos', 'site_admin': False, 'starred_url': '[https://api.github.com/users/alvr-org/starred{/owner}{/repo}](https://api.github.com/users/alvr-org/starred%7B/owner%7D%7B/repo%7D)', 'subscriptions_url': 'https://api.github.com/users/alvr-org/subscriptions', 'type': 'Organization', 'url': 'https://api.github.com/users/alvr-org'}}, 'html_url': 'https://github.com/alvr-org/PhoneVR/pull/376', 'id': 2050536757, 'issue_url': 'https://api.github.com/repos/alvr-org/PhoneVR/issues/376', 'labels': [], 'locked': False, 'maintainer_can_modify': False, 'merge_commit_sha': 'e593cba887ddefb4b712fe0eabcf1c0de55fecac', 'mergeable': None, 'mergeable_state': 'unknown', 'merged': False, 'merged_at': None, 'merged_by': None, 'milestone': None, 'node_id': 'PR_kwDOEMr8pM56OLU1', 'number': 376, 'patch_url': 'https://github.com/alvr-org/PhoneVR/pull/376.patch', 'rebaseable': None, 'requested_reviewers': [], 'requested_teams': [], 'review_comment_url': '[https://api.github.com/repos/alvr-org/PhoneVR/pulls/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/pulls/comments%7B/number%7D)', 'review_comments': 0, 'review_comments_url': 'https://api.github.com/repos/alvr-org/PhoneVR/pulls/376/comments', 'state': 'open', 'statuses_url': 'https://api.github.com/repos/alvr-org/PhoneVR/statuses/2ea57f71dfdd7c9c5be44eba3fa31a15667d4973', 'title': 'Test ci debug', 'updated_at': '2024-09-03T07:07:02Z', 'url': 'https://api.github.com/repos/alvr-org/PhoneVR/pulls/376', 'user': {'avatar_url': 'https://avatars.githubusercontent.com/u/4137788?v=4', 'events_url': '[https://api.github.com/users/ShootingKing-AM/events{/privacy}](https://api.github.com/users/ShootingKing-AM/events%7B/privacy%7D)', 'followers_url': 'https://api.github.com/users/ShootingKing-AM/followers', 'following_url': '[https://api.github.com/users/ShootingKing-AM/following{/other_user}](https://api.github.com/users/ShootingKing-AM/following%7B/other_user%7D)', 'gists_url': '[https://api.github.com/users/ShootingKing-AM/gists{/gist_id}](https://api.github.com/users/ShootingKing-AM/gists%7B/gist_id%7D)', 'gravatar_id': '', 'html_url': 'https://github.com/ShootingKing-AM', 'id': 4137788, 'login': 'ShootingKing-AM', 'node_id': 'MDQ6VXNlcjQxMzc3ODg=', 'organizations_url': 'https://api.github.com/users/ShootingKing-AM/orgs', 'received_events_url': 'https://api.github.com/users/ShootingKing-AM/received_events', 'repos_url': 'https://api.github.com/users/ShootingKing-AM/repos', 'site_admin': False, 'starred_url': '[https://api.github.com/users/ShootingKing-AM/starred{/owner}{/repo}](https://api.github.com/users/ShootingKing-AM/starred%7B/owner%7D%7B/repo%7D)', 'subscriptions_url': 'https://api.github.com/users/ShootingKing-AM/subscriptions', 'type': 'User', 'url': 'https://api.github.com/users/ShootingKing-AM'}}, 'repository': {'allow_forking': True, 'archive_url': '[https://api.github.com/repos/alvr-org/PhoneVR/{archive_format}{/ref}](https://api.github.com/repos/alvr-org/PhoneVR/%7Barchive_format%7D%7B/ref%7D)', 'archived': False, 'assignees_url': '[https://api.github.com/repos/alvr-org/PhoneVR/assignees{/user}](https://api.github.com/repos/alvr-org/PhoneVR/assignees%7B/user%7D)', 'blobs_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/blobs{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/blobs%7B/sha%7D)', 'branches_url': '[https://api.github.com/repos/alvr-org/PhoneVR/branches{/branch}](https://api.github.com/repos/alvr-org/PhoneVR/branches%7B/branch%7D)', 'clone_url': 'https://github.com/alvr-org/PhoneVR.git', 'collaborators_url': '[https://api.github.com/repos/alvr-org/PhoneVR/collaborators{/collaborator}](https://api.github.com/repos/alvr-org/PhoneVR/collaborators%7B/collaborator%7D)', 'comments_url': '[https://api.github.com/repos/alvr-org/PhoneVR/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/comments%7B/number%7D)', 'commits_url': '[https://api.github.com/repos/alvr-org/PhoneVR/commits{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/commits%7B/sha%7D)', 'compare_url': '[https://api.github.com/repos/alvr-org/PhoneVR/compare/{base}...{head}](https://api.github.com/repos/alvr-org/PhoneVR/compare/%7Bbase%7D...%7Bhead%7D)', 'contents_url': '[https://api.github.com/repos/alvr-org/PhoneVR/contents/{+path}](https://api.github.com/repos/alvr-org/PhoneVR/contents/%7B+path%7D)', 'contributors_url': 'https://api.github.com/repos/alvr-org/PhoneVR/contributors', 'created_at': '2020-07-22T17:13:03Z', 'custom_properties': {}, 'default_branch': 'master', 'deployments_url': 'https://api.github.com/repos/alvr-org/PhoneVR/deployments', 'description': 'Use Steam VR-enabled applications with your phone as HMD (Head-mounted display). The only Open-Source solution to similar commercial packages like VRidge, iVRy, Trinus etc etc.', 'disabled': False, 'downloads_url': 'https://api.github.com/repos/alvr-org/PhoneVR/downloads', 'events_url': 'https://api.github.com/repos/alvr-org/PhoneVR/events', 'fork': False, 'forks': 38, 'forks_count': 38, 'forks_url': 'https://api.github.com/repos/alvr-org/PhoneVR/forks', 'full_name': 'alvr-org/PhoneVR', 'git_commits_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/commits{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/commits%7B/sha%7D)', 'git_refs_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/refs{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/refs%7B/sha%7D)', 'git_tags_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/tags{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/tags%7B/sha%7D)', 'git_url': 'git://github.com/alvr-org/PhoneVR.git', 'has_discussions': False, 'has_downloads': True, 'has_issues': True, 'has_pages': False, 'has_projects': True, 'has_wiki': True, 'homepage': '', 'hooks_url': 'https://api.github.com/repos/alvr-org/PhoneVR/hooks', 'html_url': 'https://github.com/alvr-org/PhoneVR', 'id': 281738404, 'is_template': False, 'issue_comment_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues/comments{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues/comments%7B/number%7D)', 'issue_events_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues/events{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues/events%7B/number%7D)', 'issues_url': '[https://api.github.com/repos/alvr-org/PhoneVR/issues{/number}](https://api.github.com/repos/alvr-org/PhoneVR/issues%7B/number%7D)', 'keys_url': '[https://api.github.com/repos/alvr-org/PhoneVR/keys{/key_id}](https://api.github.com/repos/alvr-org/PhoneVR/keys%7B/key_id%7D)', 'labels_url': '[https://api.github.com/repos/alvr-org/PhoneVR/labels{/name}](https://api.github.com/repos/alvr-org/PhoneVR/labels%7B/name%7D)', 'language': 'C++', 'languages_url': 'https://api.github.com/repos/alvr-org/PhoneVR/languages', 'license': {'key': 'gpl-3.0', 'name': 'GNU General Public License v3.0', 'node_id': 'MDc6TGljZW5zZTk=', 'spdx_id': 'GPL-3.0', 'url': 'https://api.github.com/licenses/gpl-3.0'}, 'merges_url': 'https://api.github.com/repos/alvr-org/PhoneVR/merges', 'milestones_url': '[https://api.github.com/repos/alvr-org/PhoneVR/milestones{/number}](https://api.github.com/repos/alvr-org/PhoneVR/milestones%7B/number%7D)', 'mirror_url': None, 'name': 'PhoneVR', 'node_id': 'MDEwOlJlcG9zaXRvcnkyODE3Mzg0MDQ=', 'notifications_url': '[https://api.github.com/repos/alvr-org/PhoneVR/notifications{?since,all,participating}](https://api.github.com/repos/alvr-org/PhoneVR/notifications%7B?since,all,participating})', 'open_issues': 32, 'open_issues_count': 32, 'owner': {'avatar_url': 'https://avatars.githubusercontent.com/u/74990209?v=4', 'events_url': '[https://api.github.com/users/alvr-org/events{/privacy}](https://api.github.com/users/alvr-org/events%7B/privacy%7D)', 'followers_url': 'https://api.github.com/users/alvr-org/followers', 'following_url': '[https://api.github.com/users/alvr-org/following{/other_user}](https://api.github.com/users/alvr-org/following%7B/other_user%7D)', 'gists_url': '[https://api.github.com/users/alvr-org/gists{/gist_id}](https://api.github.com/users/alvr-org/gists%7B/gist_id%7D)', 'gravatar_id': '', 'html_url': 'https://github.com/alvr-org', 'id': 74990209, 'login': 'alvr-org', 'node_id': 'MDEyOk9yZ2FuaXphdGlvbjc0OTkwMjA5', 'organizations_url': 'https://api.github.com/users/alvr-org/orgs', 'received_events_url': 'https://api.github.com/users/alvr-org/received_events', 'repos_url': 'https://api.github.com/users/alvr-org/repos', 'site_admin': False, 'starred_url': '[https://api.github.com/users/alvr-org/starred{/owner}{/repo}](https://api.github.com/users/alvr-org/starred%7B/owner%7D%7B/repo%7D)', 'subscriptions_url': 'https://api.github.com/users/alvr-org/subscriptions', 'type': 'Organization', 'url': 'https://api.github.com/users/alvr-org'}, 'private': False, 'pulls_url': '[https://api.github.com/repos/alvr-org/PhoneVR/pulls{/number}](https://api.github.com/repos/alvr-org/PhoneVR/pulls%7B/number%7D)', 'pushed_at': '2024-09-03T07:07:02Z', 'releases_url': '[https://api.github.com/repos/alvr-org/PhoneVR/releases{/id}](https://api.github.com/repos/alvr-org/PhoneVR/releases%7B/id%7D)', 'size': 24280, 'ssh_url': 'git@github.com:alvr-org/PhoneVR.git', 'stargazers_count': 546, 'stargazers_url': 'https://api.github.com/repos/alvr-org/PhoneVR/stargazers', 'statuses_url': '[https://api.github.com/repos/alvr-org/PhoneVR/statuses/{sha}](https://api.github.com/repos/alvr-org/PhoneVR/statuses/%7Bsha%7D)', 'subscribers_url': 'https://api.github.com/repos/alvr-org/PhoneVR/subscribers', 'subscription_url': 'https://api.github.com/repos/alvr-org/PhoneVR/subscription', 'svn_url': 'https://github.com/alvr-org/PhoneVR', 'tags_url': 'https://api.github.com/repos/alvr-org/PhoneVR/tags', 'teams_url': 'https://api.github.com/repos/alvr-org/PhoneVR/teams', 'topics': ['android', 'asio', 'eigen', 'hmd', 'jni', 'mobile-vr', 'opengl-es', 'phone', 'phone-hmd', 'phonevr', 'smartphone', 'steamvr', 'steamvr-driver', 'virtual-reality', 'vr', 'wifi'], 'trees_url': '[https://api.github.com/repos/alvr-org/PhoneVR/git/trees{/sha}](https://api.github.com/repos/alvr-org/PhoneVR/git/trees%7B/sha%7D)', 'updated_at': '2024-09-03T02:01:39Z', 'url': 'https://api.github.com/repos/alvr-org/PhoneVR', 'visibility': 'public', 'watchers': 546, 'watchers_count': 546, 'web_commit_signoff_required': False}, 'sender': {'avatar_url': 'https://avatars.githubusercontent.com/u/4137788?v=4', 'events_url': '[https://api.github.com/users/ShootingKing-AM/events{/privacy}](https://api.github.com/users/ShootingKing-AM/events%7B/privacy%7D)', 'followers_url': 'https://api.github.com/users/ShootingKing-AM/followers', 'following_url': '[https://api.github.com/users/ShootingKing-AM/following{/other_user}](https://api.github.com/users/ShootingKing-AM/following%7B/other_user%7D)', 'gists_url': '[https://api.github.com/users/ShootingKing-AM/gists{/gist_id}](https://api.github.com/users/ShootingKing-AM/gists%7B/gist_id%7D)', 'gravatar_id': '', 'html_url': 'https://github.com/ShootingKing-AM', 'id': 4137788, 'login': 'ShootingKing-AM', 'node_id': 'MDQ6VXNlcjQxMzc3ODg=', 'organizations_url': 'https://api.github.com/users/ShootingKing-AM/orgs', 'received_events_url': 'https://api.github.com/users/ShootingKing-AM/received_events', 'repos_url': 'https://api.github.com/users/ShootingKing-AM/repos', 'site_admin': False, 'starred_url': '[https://api.github.com/users/ShootingKing-AM/starred{/owner}{/repo}](https://api.github.com/users/ShootingKing-AM/starred%7B/owner%7D%7B/repo%7D)', 'subscriptions_url': 'https://api.github.com/users/ShootingKing-AM/subscriptions', 'type': 'User', 'url': 'https://api.github.com/users/ShootingKing-AM'}}, event_file=None, event_name='pull_request', is_fork=False, repo='alvr-org/PhoneVR', commit='2ea57f71dfdd7c9c5be44eba3fa31a15667d4973', json_file=None, json_thousands_separator='\u2008', json_suite_details=False, json_test_case_results=False, fail_on_errors=True, fail_on_failures=True, action_fail=False, action_fail_on_inconclusive=False, files_glob='code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/**/TEST-*.xml\n', junit_files_glob=None, nunit_files_glob=None, xunit_files_glob=None, trx_files_glob=None, test_file_prefix=None, time_factor=1.0, check_name='Test Results', comment_title='🛠 Test Results', comment_mode='always', check_run=True, job_summary=True, compare_earlier=True, pull_request_build='merge', test_changes_limit=10, report_individual_runs=True, report_suite_out_logs=True, report_suite_err_logs=True, dedup_classes_by_file_name=True, large_files=False, ignore_runs=False, check_run_annotation=[], seconds_between_github_reads=0.25, seconds_between_github_writes=2.0, secondary_rate_limit_wait_seconds=60.0, search_pull_requests=False)
  2024-09-03 07:59:39 +0000 - publish -  INFO - Available memory to read files: 8.2 GiB
  2024-09-03 07:59:39 +0000 - publish -  INFO - Reading files code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/**/TEST-*.xml (2 files, 19.3 KiB)
  2024-09-03 07:59:39 +0000 - publish - DEBUG - reading files ['code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/gvr/TEST-Pixel_2(AVD) - 11-_app-gvr.xml', 'code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/noGvr/TEST-Pixel_2(AVD) - 11-_app-noGvr.xml']
  2024-09-03 07:59:39 +0000 - publish -  INFO - Detected 2 JUnit XML files (19.3 KiB)
  2024-09-03 07:59:39 +0000 - publish - DEBUG - detected JUnit XML files ['code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/gvr/TEST-Pixel_2(AVD) - 11-_app-gvr.xml', 'code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/noGvr/TEST-Pixel_2(AVD) - 11-_app-noGvr.xml']
  2024-09-03 07:59:39 +0000 - publish -  INFO - Finished reading 2 files in 0.00 seconds
  2024-09-03 07:59:39 +0000 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.github.com:443
  2024-09-03 07:59:39 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR HTTP/11" 200 None
  2024-09-03 07:59:39 +0000 - publish -  INFO - Publishing failure results for commit 2ea57f71dfdd7c9c5be44eba3fa31a15667d4973
  2024-09-03 07:59:39 +0000 - publish - DEBUG - Publishing UnitTestRunResults(files=2, errors=[], suites=2, duration=1725350141, suite_details=[UnitTestSuite(name=None, tests=0, skipped=0, failures=0, errors=0, stdout=None, stderr='Exception thrown during onBeforeAll invocation of plugin AndroidTestApkInstallerPlugin: ErrorName: Test APK installation Error\nNameSpace: AndroidTestApkInstallerPlugin\nErrorCode: 2002\nErrorType: TEST\nMessage: Failed to install APK: [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/gvr/debug/PhoneVR-v2.0.0-beta-gvr-debug.apk] on device emulator-5554.\nFailed to install APK: [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/gvr/debug/PhoneVR-v2.0.0-beta-gvr-debug.apk] on device emulator-5554.\ncom.google.testing.platform.core.error.UtpException: ErrorName: Test APK installation Error\nNameSpace: AndroidTestApkInstallerPlugin\nErrorCode: 2002\nErrorType: TEST\nMessage: Failed to install APK: [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/gvr/debug/PhoneVR-v2.0.0-beta-gvr-debug.apk] on device emulator-5554.\n\tat com.android.tools.utp.plugins.host.apkinstaller.AndroidTestApkInstallerPlugin.beforeAll(AndroidTestApkInstallerPlugin.kt:231)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1$1.invoke(PluginLifecycle.kt:206)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1$1.invoke(PluginLifecycle.kt:206)\n\tat com.google.testing.platform.plugin.PluginLifecycle$verify$1.invoke(PluginLifecycle.kt:131)\n\tat com.google.testing.platform.api.event.Events$DefaultImpls.verify(Events.kt:38)\n\tat com.google.testing.platform.core.event.MutableEvents$DefaultImpls.verify(MutableEvents.kt:7)\n\tat com.google.testing.platform.core.event.PhaseGlobalEvents.verify(PhaseGlobalEvents.kt:20)\n\tat com.google.testing.platform.plugin.PluginLifecycle.verify(PluginLifecycle.kt:131)\n\tat com.google.testing.platform.plugin.PluginLifecycle.verify$default(PluginLifecycle.kt:127)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1.invoke(PluginLifecycle.kt:206)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1.invoke(PluginLifecycle.kt:205)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt$invokeOrThrow$1$1.invoke(PluginLifecycle.kt:538)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt$invokeOrThrow$1$1.invoke(PluginLifecycle.kt:538)\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\n\tat com.google.testing.platform.core.telemetry.SequentialEventRecordRequest.record$java_com_google_testing_platform_core_telemetry_telemetry_api(EventRecordRequest.kt:71)\n\tat com.google.testing.platform.core.telemetry.DiagnosticsExtKt.record(DiagnosticsExt.kt:27)\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.createEvent(Telemetry.kt:60)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow(PluginLifecycle.kt:533)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow$default(PluginLifecycle.kt:517)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1.invoke(PluginLifecycle.kt:205)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1.invoke(PluginLifecycle.kt:199)\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\n\tat com.google.testing.platform.core.telemetry.SequentialEventRecordRequest.record$java_com_google_testing_platform_core_telemetry_telemetry_api(EventRecordRequest.kt:71)\n\tat com.google.testing.platform.core.telemetry.DiagnosticsExtKt.record(DiagnosticsExt.kt:27)\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.createEvent(Telemetry.kt:60)\n\tat com.google.testing.platform.plugin.PluginLifecycle.onBeforeAll(PluginLifecycle.kt:197)\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$5.invoke(SingleDeviceExecutor.kt:136)\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$5.invoke(SingleDeviceExecutor.kt:136)\n\tat com.google.testing.platform.lib.cancellation.ProcessCancellationContext.runUnlessCancelled(ProcessCancellationContext.kt:136)\n\tat com.google.testing.platform.executor.SingleDeviceExecutor.execute(SingleDeviceExecutor.kt:136)\n\tat com.google.testing.platform.RunnerImpl.run(RunnerImpl.kt:121)\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$4$2.invoke(NonInteractiveServerStrategy.kt:98)\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$4$2.invoke(NonInteractiveServerStrategy.kt:98)\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\n\tat com.google.testing.platform.core.telemetry.SequentialEventRecordRequest.record$java_com_google_testing_platform_core_telemetry_telemetry_api(EventRecordRequest.kt:71)\n\tat com.google.testing.platform.core.telemetry.DiagnosticsExtKt.record(DiagnosticsExt.kt:27)\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.createEvent(Telemetry.kt:60)\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy.run(NonInteractiveServerStrategy.kt:95)\n\tat com.google.testing.platform.main.MainKt$main$4.invokeSuspend(Main.kt:75)\n\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\n\tat kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)\n\tat kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)\n\tat kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)\n\tat kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)\n\tat kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)\n\tat com.google.testing.platform.main.MainKt.main(Main.kt:73)\n\tat com.google.testing.platform.main.MainKt.main$default(Main.kt:35)\n\tat com.google.testing.platform.main.MainKt.main(Main.kt)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n\tat com.google.testing.platform.launcher.Launcher.main(Launcher.java:154)'), UnitTestSuite(name='viritualisres.phonevr.ALVRActivityTest', tests=2, skipped=0, failures=1, errors=0, stdout=None, stderr=None)], tests=2, tests_succ=1, tests_skip=0, tests_fail=1, tests_error=0, runs=2, runs_succ=1, runs_skip=0, runs_fail=1, runs_error=0, commit='2ea57f71dfdd7c9c5be44eba3fa31a15667d4973')
  2024-09-03 07:59:39 +0000 - publish - DEBUG - comparing against before=3d01fcf1cc198e24b45cdc610dd0735151a047ed
  2024-09-03 07:59:40 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR/commits/3d01fcf1cc198e24b45cdc610dd0735151a047ed HTTP/11" 200 None
  2024-09-03 07:59:41 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR/commits/3d01fcf1cc198e24b45cdc610dd0735151a047ed/check-runs?per_page=1 HTTP/11" 200 None
  2024-09-03 07:59:41 +0000 - publish - DEBUG - found 0 check runs for commit 3d01fcf1cc198e24b45cdc610dd0735151a047ed
  2024-09-03 07:59:41 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR/commits/3d01fcf1cc198e24b45cdc610dd0735151a047ed/check-runs?per_page=100 HTTP/11" 200 None
  2024-09-03 07:59:41 +0000 - publish - DEBUG - there are 0 check runs with title Test Results
  2024-09-03 07:59:41 +0000 - publish - DEBUG - stats with delta: UnitTestRunResults(files=2, errors=[], suites=2, duration=1725350141, suite_details=[UnitTestSuite(name=None, tests=0, skipped=0, failures=0, errors=0, stdout=None, stderr='Exception thrown during onBeforeAll invocation of plugin AndroidTestApkInstallerPlugin: ErrorName: Test APK installation Error\nNameSpace: AndroidTestApkInstallerPlugin\nErrorCode: 2002\nErrorType: TEST\nMessage: Failed to install APK: [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/gvr/debug/PhoneVR-v2.0.0-beta-gvr-debug.apk] on device emulator-5554.\nFailed to install APK: [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/gvr/debug/PhoneVR-v2.0.0-beta-gvr-debug.apk] on device emulator-5554.\ncom.google.testing.platform.core.error.UtpException: ErrorName: Test APK installation Error\nNameSpace: AndroidTestApkInstallerPlugin\nErrorCode: 2002\nErrorType: TEST\nMessage: Failed to install APK: [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/gvr/debug/PhoneVR-v2.0.0-beta-gvr-debug.apk] on device emulator-5554.\n\tat com.android.tools.utp.plugins.host.apkinstaller.AndroidTestApkInstallerPlugin.beforeAll(AndroidTestApkInstallerPlugin.kt:231)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1$1.invoke(PluginLifecycle.kt:206)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1$1.invoke(PluginLifecycle.kt:206)\n\tat com.google.testing.platform.plugin.PluginLifecycle$verify$1.invoke(PluginLifecycle.kt:131)\n\tat com.google.testing.platform.api.event.Events$DefaultImpls.verify(Events.kt:38)\n\tat com.google.testing.platform.core.event.MutableEvents$DefaultImpls.verify(MutableEvents.kt:7)\n\tat com.google.testing.platform.core.event.PhaseGlobalEvents.verify(PhaseGlobalEvents.kt:20)\n\tat com.google.testing.platform.plugin.PluginLifecycle.verify(PluginLifecycle.kt:131)\n\tat com.google.testing.platform.plugin.PluginLifecycle.verify$default(PluginLifecycle.kt:127)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1.invoke(PluginLifecycle.kt:206)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1$2$1.invoke(PluginLifecycle.kt:205)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt$invokeOrThrow$1$1.invoke(PluginLifecycle.kt:538)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt$invokeOrThrow$1$1.invoke(PluginLifecycle.kt:538)\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\n\tat com.google.testing.platform.core.telemetry.SequentialEventRecordRequest.record$java_com_google_testing_platform_core_telemetry_telemetry_api(EventRecordRequest.kt:71)\n\tat com.google.testing.platform.core.telemetry.DiagnosticsExtKt.record(DiagnosticsExt.kt:27)\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.createEvent(Telemetry.kt:60)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow(PluginLifecycle.kt:533)\n\tat com.google.testing.platform.plugin.PluginLifecycleKt.invokeOrThrow$default(PluginLifecycle.kt:517)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1.invoke(PluginLifecycle.kt:205)\n\tat com.google.testing.platform.plugin.PluginLifecycle$onBeforeAll$1$1.invoke(PluginLifecycle.kt:199)\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\n\tat com.google.testing.platform.core.telemetry.SequentialEventRecordRequest.record$java_com_google_testing_platform_core_telemetry_telemetry_api(EventRecordRequest.kt:71)\n\tat com.google.testing.platform.core.telemetry.DiagnosticsExtKt.record(DiagnosticsExt.kt:27)\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.createEvent(Telemetry.kt:60)\n\tat com.google.testing.platform.plugin.PluginLifecycle.onBeforeAll(PluginLifecycle.kt:197)\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$5.invoke(SingleDeviceExecutor.kt:136)\n\tat com.google.testing.platform.executor.SingleDeviceExecutor$execute$5.invoke(SingleDeviceExecutor.kt:136)\n\tat com.google.testing.platform.lib.cancellation.ProcessCancellationContext.runUnlessCancelled(ProcessCancellationContext.kt:136)\n\tat com.google.testing.platform.executor.SingleDeviceExecutor.execute(SingleDeviceExecutor.kt:136)\n\tat com.google.testing.platform.RunnerImpl.run(RunnerImpl.kt:121)\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$4$2.invoke(NonInteractiveServerStrategy.kt:98)\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy$run$4$2.invoke(NonInteractiveServerStrategy.kt:98)\n\tat com.google.testing.platform.core.telemetry.common.noop.NoopDiagnosticsScope.recordEvent(NoopDiagnosticsScope.kt:35)\n\tat com.google.testing.platform.core.telemetry.SequentialEventRecordRequest.record$java_com_google_testing_platform_core_telemetry_telemetry_api(EventRecordRequest.kt:71)\n\tat com.google.testing.platform.core.telemetry.DiagnosticsExtKt.record(DiagnosticsExt.kt:27)\n\tat com.google.testing.platform.core.telemetry.TelemetryKt.createEvent(Telemetry.kt:60)\n\tat com.google.testing.platform.server.strategy.NonInteractiveServerStrategy.run(NonInteractiveServerStrategy.kt:95)\n\tat com.google.testing.platform.main.MainKt$main$4.invokeSuspend(Main.kt:75)\n\tat kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)\n\tat kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)\n\tat kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)\n\tat kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)\n\tat kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)\n\tat kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)\n\tat kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)\n\tat com.google.testing.platform.main.MainKt.main(Main.kt:73)\n\tat com.google.testing.platform.main.MainKt.main$default(Main.kt:35)\n\tat com.google.testing.platform.main.MainKt.main(Main.kt)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n\tat com.google.testing.platform.launcher.Launcher.main(Launcher.java:154)'), UnitTestSuite(name='viritualisres.phonevr.ALVRActivityTest', tests=2, skipped=0, failures=1, errors=0, stdout=None, stderr=None)], tests=2, tests_succ=1, tests_skip=0, tests_fail=1, tests_error=0, runs=2, runs_succ=1, runs_skip=0, runs_fail=1, runs_error=0, commit='2ea57f71dfdd7c9c5be44eba3fa31a15667d4973')
  2024-09-03 07:59:41 +0000 - publish - DEBUG - creating check with 2 annotations
  2024-09-03 07:59:42 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "POST /repos/alvr-org/PhoneVR/check-runs HTTP/11" 422 202
  Traceback (most recent call last):
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/v2/macos/../python/publish_test_results.py", line 546, in <module>
      main(settings, gha)
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/v2/macos/../python/publish_test_results.py", line 269, in main
      Publisher(settings, gh, gha).publish(stats, results.case_results, conclusion)
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/v2/python/publish/publisher.py", line 233, in publish
      data = self.publish_check(data)
             ^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/v2/python/publish/publisher.py", line 461, in publish_check
      check_run = self._repo.create_check_run(name=self._settings.check_name,
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/PhoneVR/PhoneVR/enricomi-publish-action-venv/lib/python3.12/site-packages/github/Repository.py", line 3793, in create_check_run
      headers, data = self._requester.requestJsonAndCheck(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/PhoneVR/PhoneVR/enricomi-publish-action-venv/lib/python3.12/site-packages/github/Requester.py", line 537, in requestJsonAndCheck
      return self.__check(*self.requestJson(verb, url, parameters, headers, input, self.__customConnection(url)))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/PhoneVR/PhoneVR/enricomi-publish-action-venv/lib/python3.12/site-packages/github/Requester.py", line [598](https://github.com/alvr-org/PhoneVR/actions/runs/10678178487/job/29594710345#step:20:607), in __check
      raise self.createException(status, responseHeaders, data)
  github.GithubException.GithubException: 422 {"message": "Validation Failed", "errors": [{"resource": "CheckRun", "code": "invalid", "field": "annotations"}], "documentation_url": "https://docs.github.com/rest/checks/runs#create-a-check-run", "status": "422"}
  Error: Process completed with exit code 1.
  ```

> Please use the following action to debug this issue further:
> 
> ```
> uses: EnricoMi/publish-unit-test-result-action/macos@debug-checkrun-anotations
> ```

i re-ran with this 10hrs ago (15mins after you last suggested the debug version of action.
run link: https://github.com/alvr-org/PhoneVR/actions/runs/10683954200/job/29613219448?pr=376
```
2024-09-03 14:17:27 +0000 - publish - DEBUG - action triggered by 'pull_request' event
  2024-09-03 14:17:27 +0000 - publish - DEBUG - Settings: Settings(token='***', actor='github-actions', api_url='https://api.github.com/', graphql_url='https://api.github.com/graphql', api_retries=10, event={'action': 'synchronize', 'after': '58261cbf67756b858a31619ca59b148b0ff2b14d', 'before': '2ea57f71dfdd7c9c5be44eba3fa31a15667d4973', 'number': 376, 'organization': {'avatar_url': 'https://avatars.githubusercontent.com/u/74990209?v=4', 'description': '', 'events_url': 'https://api.github.com/orgs/alvr-org/events', 'hooks_url': 'https://api.github.com/orgs/alvr-org/hooks', 'id': 74990209, 'issues_url': 'https://api.github.com/orgs/alvr-org/issues', 'login': 'alvr-org', 'members_url': '[https://api.github.com/orgs/alvr-org/members{/member}](https://api.github.com/orgs/alvr-org/members%7B/member%7D)', 'node_id': 'MDEyOk9yZ2FuaXphdGlvbjc0OTkwMjA5', 'public_members_url': '[https://api.github.com/orgs/alvr-org/public_members{/member}](https://api.github.com/orgs/alvr-org/public_members%7B/member%7D)', 'repos_url': 'https://api.github.com/orgs/alvr-org/repos', 'url': 'https://api.github.com/orgs/alvr-org'}, 'pull_request': {'_links': {'comment
  2024-09-03 14:17:27 +0000 - publish -  INFO - Available memory to read files: 7.6 GiB
  2024-09-03 14:17:27 +0000 - publish -  INFO - Reading files code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/**/TEST-*.xml (2 files, 15.9 KiB)
  2024-09-03 14:17:27 +0000 - publish - DEBUG - reading files ['code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/noGvr/TEST-Pixel_2(AVD) - 11-_app-noGvr.xml', 'code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/gvr/TEST-Pixel_2(AVD) - 11-_app-gvr.xml']
  2024-09-03 14:17:27 +0000 - publish -  INFO - Detected 2 JUnit XML files (15.9 KiB)
  2024-09-03 14:17:27 +0000 - publish - DEBUG - detected JUnit XML files ['code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/noGvr/TEST-Pixel_2(AVD) - 11-_app-noGvr.xml', 'code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/gvr/TEST-Pixel_2(AVD) - 11-_app-gvr.xml']
  2024-09-03 14:17:27 +0000 - publish -  INFO - Finished reading 2 files in 0.00 seconds
  2024-09-03 14:17:27 +0000 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.github.com:443
  2024-09-03 14:17:27 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR HTTP/11" 200 None
  2024-09-03 14:17:27 +0000 - publish -  INFO - Publishing failure results for commit 58261cbf67756b858a31619ca59b148b0ff2b14d
  2024-09-03 14:17:27 +0000 - publish - DEBUG - Publishing UnitTestRunResults(files=2, errors=[], suites=2, duration=1725373658, suite_details=[UnitTestSuite(name=None, tests=0, skipped=0, failures=0, errors=0, stdout=None, stderr='Exception thrown during onBeforeAll invocation of plugin AndroidTestApkInstallerPlugin: ErrorName: UNKNOWN\nNameSpace: DdmlibAndroidDeviceController\nErrorCode: 1\nErrorType: TEST\nMessage: Failed to install split APK(s): [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/noGvr/debug/PhoneVR-v2.0.0-beta-noGvr-debug.apk]\nFailed to install split APK(s): [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/noGvr/debug/PhoneVR-v2.0.0-beta-noGvr-debug.apk]\nFailed to commit install session 540840919 with command package install-commit 540840919. Error: Unknown failure: cmd: Failure calling service package: Broken pipe (32)\ncom.android.ddmlib.InstallException: Failed to commit install session 540840919 with command packa
  2024-09-03 14:17:27 +0000 - publish - DEBUG - comparing against before=2ea57f71dfdd7c9c5be44eba3fa31a15667d4973
  2024-09-03 14:17:28 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR/commits/2ea57f71dfdd7c9c5be44eba3fa31a15667d4973 HTTP/11" 200 None
  2024-09-03 14:17:28 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR/commits/2ea57f71dfdd7c9c5be44eba3fa31a15667d4973/check-runs?per_page=1 HTTP/11" 200 None
  2024-09-03 14:17:28 +0000 - publish - DEBUG - found 6 check runs for commit 2ea57f71dfdd7c9c5be44eba3fa31a15667d4973
  2024-09-03 14:17:29 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "GET /repos/alvr-org/PhoneVR/commits/2ea57f71dfdd7c9c5be44eba3fa31a15667d4973/check-runs?per_page=100 HTTP/11" 200 None
  2024-09-03 14:17:29 +0000 - publish - DEBUG - there are 0 check runs with title Test Results
  2024-09-03 14:17:29 +0000 - publish - DEBUG - stats with delta: UnitTestRunResults(files=2, errors=[], suites=2, duration=1725373658, suite_details=[UnitTestSuite(name=None, tests=0, skipped=0, failures=0, errors=0, stdout=None, stderr='Exception thrown during onBeforeAll invocation of plugin AndroidTestApkInstallerPlugin: ErrorName: UNKNOWN\nNameSpace: DdmlibAndroidDeviceController\nErrorCode: 1\nErrorType: TEST\nMessage: Failed to install split APK(s): [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/noGvr/debug/PhoneVR-v2.0.0-beta-noGvr-debug.apk]\nFailed to install split APK(s): [/Users/runner/work/PhoneVR/PhoneVR/code/mobile/android/PhoneVR/app/build/outputs/apk/noGvr/debug/PhoneVR-v2.0.0-beta-noGvr-debug.apk]\nFailed to commit install session 540840919 with command package install-commit 540840919. Error: Unknown failure: cmd: Failure calling service package: Broken pipe (32)\ncom.android.ddmlib.InstallException: Failed to commit install session 540840919 with comman
  2024-09-03 14:17:29 +0000 - publish - DEBUG - creating check with 2 annotations:
  2024-09-03 14:17:29 +0000 - publish - DEBUG - [{'path': 'viritualisres.phonevr.ALVRActivityTest', 'start_line': 0, 'end_line': 0, 'annotation_level': 'warning', 'message': 'code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/connected/debug/flavors/gvr/TEST-Pixel_2(AVD) - 11-_app-gvr.xml\u2003[took 1m 45s]', 'title': 'saveDeviceScreenBitmap (viritualisres.phonevr.ALVRActivityTest) failed', 'raw_details': "androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: view.getId() is <2131296742/viritualisres.phonevr:id/ui_settings_button>\n\nView Hierarchy:\n+>DecorView{id=-1, visibility=VISIBLE, width=1080, height=1920, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params={(0,0)(fillxfill) ty=BASE_APPLICATION wanim=0x10302fe\nfl=LAYOUT_IN_SCREEN LAYOUT_INSET_DECOR SPLIT_TOUCH HARDWARE_ACCELERATED DRAWS_SYSTEM_BAR_BACKGROUNDS\npfl=FO
  2024-09-03 14:17:30 +0000 - urllib3.connectionpool - DEBUG - [https://api.github.com:443](https://api.github.com/) "POST /repos/alvr-org/PhoneVR/check-runs HTTP/11" 422 202
  Traceback (most recent call last):
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/debug-checkrun-anotations/macos/../python/publish_test_results.py", line 546, in <module>
      main(settings, gha)
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/debug-checkrun-anotations/macos/../python/publish_test_results.py", line 269, in main
      Publisher(settings, gh, gha).publish(stats, results.case_results, conclusion)
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/debug-checkrun-anotations/python/publish/publisher.py", line 233, in publish
      data = self.publish_check(data)
             ^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/_actions/EnricoMi/publish-unit-test-result-action/debug-checkrun-anotations/python/publish/publisher.py", line 462, in publish_check
      check_run = self._repo.create_check_run(name=self._settings.check_name,
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/PhoneVR/PhoneVR/enricomi-publish-action-venv/lib/python3.12/site-packages/github/Repository.py", line 3793, in create_check_run
      headers, data = self._requester.requestJsonAndCheck(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/PhoneVR/PhoneVR/enricomi-publish-action-venv/lib/python3.12/site-packages/github/Requester.py", line 537, in requestJsonAndCheck
      return self.__check(*self.requestJson(verb, url, parameters, headers, input, self.__customConnection(url)))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/runner/work/PhoneVR/PhoneVR/enricomi-publish-action-venv/lib/python3.12/site-packages/github/Requester.py", line [598](https://github.com/alvr-org/PhoneVR/actions/runs/10683954200/job/29613219448?pr=376#step:20:607), in __check
      raise self.createException(status, responseHeaders, data)
  github.GithubException.GithubException: 422 {"message": "Validation Failed", "errors": [{"resource": "CheckRun", "code": "invalid", "field": "annotations"}], "documentation_url": "https://docs.github.com/rest/checks/runs#create-a-check-run", "status": "422"}
  Error: Process completed with exit code 1.
  ```
  
> Issue fixed, please rerun with above debug branch.
> 
> Is there a chance to upload test result files `code/mobile/android/PhoneVR/app/build/outputs/androidTest-results/**/TEST-*.xml` as artifacts so I can inspect them?

Done. Re-run running at https://github.com/alvr-org/PhoneVR/actions/runs/10683954200/job/29642007254 just now (it will take some time 30-40mins to complete)

The test results will be available as a artifact in run/check summary, which should be after 30-40mins. 
For example, this run: https://github.com/alvr-org/PhoneVR/actions/runs/10678178487 's if you scroll down there are "Test Results" attachment. 

Btw, thanks for your quick reply.

--- a/python/publish/__init__.py
+++ b/python/publish/__init__.py
@@ -863,13 +863,13 @@ def get_case_annotations(case_results: UnitTestCaseResults,
 
 def get_error_annotation(error: ParseError) -> Annotation:
     return Annotation(
-        path=error.file,
+        path=error.file or '/',
         start_line=error.line or 0,
         end_line=error.line or 0,
         start_column=error.column,
         end_column=error.column,
         annotation_level='failure',
-        message=error.message,
+        message=error.message or '',
         title=f'Error processing result file',
         raw_details=error.file
     )
@@ -882,14 +882,14 @@ def get_error_annotations(parse_errors: List[ParseError]) -> List[Annotation]:
 def get_suite_annotations_for_suite(suite: UnitTestSuite, with_suite_out_logs: bool, with_suite_err_logs: bool) -> List[Annotation]:
     return [
         Annotation(
-            path=suite.name,
+            path=suite.name or '/',
             start_line=0,
             end_line=0,
             start_column=None,
             end_column=None,
             annotation_level='warning' if source == 'stderr' else 'notice',
-            message=f'Test suite {suite.name} has the following {source} output (see Raw output).',
-            title=f'Logging on {source} of test suite {suite.name}',
+            message=f'Test suite {suite.name or "<unknown>"} has the following {source} output (see Raw output).',
+            title=f'Logging on {source} of test suite {suite.name or "<unknown>"}',
             raw_details=details
         )
         for details, source in ([(suite.stdout, 'stdout')] if with_suite_out_logs else []) +

